src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -939,28 +939,11 @@ private void ScanSyntaxToken(ref TokenInfo info)
 
         private void CheckFeatureAvailability(MessageID feature)
         {
-            var options = this.Options;
-            if (options.IsFeatureEnabled(feature))
-            {
-                return;
-            }
-
-            string requiredFeature = feature.RequiredFeature();
-            if (requiredFeature != null)
+            var info = feature.GetFeatureAvailabilityDiagnosticInfo(Options);
+            if (info != null)
```

Comment:
This PR seems to jump between `info != null` and `!(info is null)`. Is there a reason?

Reply:
No, not really. I was probably just editing the file at a different time and forgot what the prevailing style was. I generally like `!(info is null)`.